### PR TITLE
Single datamodel entry

### PR DESCRIPTION
**Pull request by jakobw**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "jquery": "^3.2.1",
-    "wikibase-data-model": "^4.0.0",
+    "wikibase-data-model": "https://github.com/wmde/WikibaseDataModelJavaScript.git#41c83e27f5e922f9f17ebcc030c6b699b6f6ccc4",
```
Comment (AlaaSarhan) [inline package.json]: better to wait until we merge and release there? or we need to remember to chabge it back when we release.

Comment (Ladsgroup) [reply package.json]: Yes, we are waiting for the other one to be merged first and then continue with this. Just checking if tests run. This PR and the ones are in "draft" state.

Comment (AlaaSarhan) [reply package.json]: oh I must have missed to see thebdraft state

**Comment by Ladsgroup** [comment]
> Looks good!
> 
> Let's wait until a release is made on wikibase.datamodel instead of depending on a specific commit? Unless urgent, I think following that pattern is better.

It's done now

**Pull request merged**